Certificate feature + Webdav client + Fixed version management (#15)
cta-epfl/ctadata: README.md

## Installation

```bash

> pip install 'ctadata>=0.2.6'

Reviewer: ```suggestion
$ pip install 'ctadata>=0.2.6'
```

Author: Ok, I can reverse this change if you want. I removed this because it's clear that we need to run this line and because when using the copy button, we do not have to remove this extra character manually. It's mainly a question of simplicity for me.

Reviewer: It's commonly done in one way or another, there are arguments for and against. 
In ambiguous cases, like this one, I think it's best to give the code author some leeway, not changing parts of the code to own preference in unrelated PR.

So yes, let's change back. We can discuss separately if it's best to do something different which will allow both distinct presentation, copying, and other features if any.

        print("found keys", h5py.File(url.split("/")[-1]).keys())
```

# Webdav Client

> To offer a smoother usage, `ctadata` offers a `webdav4` client, see [webdav4](https://github.com/skshetry/webdav4) for documentation.

Reviewer: I am not sure if it's smoother, but anyway it's better to avoid disputable style qualifiers and say directly what is different about this alternative.

```suggestion
In order to make use of bare WebDAV interface of the storage, `ctadata` also provides a configured `webdav4` client (see [webdav4](https://github.com/skshetry/webdav4) for documentation).
```

```python
client = ctadata.webdav4_client()
client.ls("/")
client.uploadFile("example.txt", "remote/example.txt")
```

Please see [WebDAV4 documenation](https://skshetry.github.io/webdav4/) for details on it's wide range of features.

> # Uploading a certificate to access the webdav

Reviewer: ```suggestion
# Delegating a proxy grid certificate to the Platform

In order to make use of your own grid certificate to access CTA-CSCS storage from within CTA interactive platform it is necessary to upload you short-term proxy certificate to the platform. `cta-data` provides an easy way to do this:
```

This tools also offers a way to upload your own time limited certificate to access the background webdav server.

```python
import ctadata
ctadata.upload_certificate('yourcertificate.crt')

> ```

Reviewer: ```suggestion
Note that if you do not upload your own certificate, you can ask to make use of a shared robot certificate used for data syncing.
```
